### PR TITLE
NN-3003 replace pagination service

### PR DESCRIPTION
**Pull request by JamieDraperUK**

### CODE REVIEW

File: backend/tests/paginationService.test.js [hunk 1]
```diff
@@ -1,48 +1,182 @@
 const service = require('../services/paginationService')
 
-describe('Pagination service', () => {
-  it('should return pages, previous, next and results when more than one page', () => {
-    const response = service.getPagination(100, 0, 20, new URL('http://localhost/'))
+describe('Pagination service 2', () => {
```
Comment (Nimphal) [inline backend/tests/paginationService.test.js:3]: Definitely non-consequential, but I think this shouldn't have the 2 in it?

**Review by Nimphal — COMMENTED** [review]
Does this also work on the visits history page? I think that's the only one we have which uses the new pagination approach.

**Review by Nimphal — APPROVED** [review]
LGTM!

**Comment by JamieDraperUK** [comment]
Good question, I'll check visits history page now @Nimphal

**Pull request merged**